src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6540,7 +6540,7 @@ void UpdateSysSizing(EnergyPlusData &state, Constant::CallIndicator const CallIn
             }
```

Comment:
If you did combine the enums as I suggested you would need a default here that did nothing or asserted(false).

Reply:
Maybe that's what I didn't like about a single enum.